src/Packagist/Api/Client.php

```diff
@@ -98,23 +99,25 @@ public function get(string $package)
     }
 
     /**
-     * Similar to {@link get()}, but uses composer metadata which is Packagist's preferred
-     * way of retrieving details, since responses are cached efficiently as static files
-     * by the Packagist service. The response lacks some metadata that is provided
-     * by {@link get()}, see https://packagist.org/apidoc for details.
-     *
-     * Caution: Returns an array of packages, you need to select the correct one
-     * from the indexed array.
-     *
-     * @since 1.6
-     *
+     * @see https://packagist.org/apidoc#get-package-data
+     * Contains tagged releases and dev versions
      * @param string $package Full qualified name ex : myname/mypackage
-     *
-     * @return \Packagist\Api\Result\Package[] An array of packages, including the requested one.
+     * @return Package[] An array of packages, including the requested one.
+     */
+    public function getComposer(string $package): array
+    {
+        return $this->respond(sprintf($this->url('/p2/%s~dev.json'), $package));
+    }
+
+    /**
+     * @see https://packagist.org/apidoc#get-package-data
+     * Contains only tagged releases
+     * @param string $package Full qualified name ex : myname/mypackage
+     * @return Package[] An array of packages, including the requested one.
      */
-    public function getComposer($package)
+    public function getComposerLite(string $package): array
     {
-        return $this->respond(sprintf($this->url('/p/%s.json'), $package));
+        return $this->respond(sprintf($this->url('/p2/%s.json'), $package));
```

Comment:
Just confirming that the change of the API we consume in this method was deliberate? I have no problem with changing the underlying API we consume from v1 to v2. Is the intention of adding `getComposerLite()` to allow access to the original v1 API endpoint, or to return the same results as `getComposer()` except without dev dependencies?

Reply:
Yes! I confirm! 
I add `getComposerLite()` to return the same results as `getComposer()` except without dev dependencies ad descripted [here](https://packagist.org/apidoc) in the `Getting package data` section.